man/man1/ice2slice.1

```diff
@@ -0,0 +1,82 @@
+.TH ice2slice 1
+
+.SH NAME
+
+ice2slice - The Ice to Slice compiler.
+
+.SH SYNOPSIS
+
+ice2slice [options] [files]
+
+.SH DESCRIPTION
+
+ice2slice compiler transforms Slice definitions in (.ice) files used by Ice to Slice definitions in (.slice) files used
```

Comment:
I'd write:
The ice2slice compiler transforms Slice definitions in `.ice` files into Slice definitions in `.slice` files. 

`.ice` files are understood by the Slice compilers provided by Ice, while `.slice` files are understood by the new Slice compiler developed as part of IceRPC.